src/pages/keyword-generator/KeywordGenerator.vue

```diff
@@ -35,187 +35,49 @@ const singleWords = computed(() => {
     return filteredKeywords;
 });
 
-// Computed property to generate n-grams based on selected n values
+// Computed property to generate n-grams based on selected n value
 const computedNgrams = computed(() => {
-  const sortedNgrams = selectedNgrams.value
-    .map(n => parseInt(n))
-    .sort((a, b) => a - b); // Sort selected n-grams in ascending order
-
-  const ngrams = sortedNgrams.map(n => {
-    const ngramList = ngramGenerate(singleWords.value, n);
-    return { n: n.toString(), list: ngramList };
-  });
-  return ngrams;
-});
-
-const showKeywords = computed(() => {
-    return computedNgrams.value.length > 0;
+  return ngramGenerate(singleWords.value, Number(selectedNgram.value));
 });
-
-// Handler for checkbox changes
-const handleCheckboxChange = (event) => {
-  const { value, checked } = event.target;
-
-  if (checked) {
-    selectedNgrams.value.push(value);
-  } else {
-    selectedNgrams.value = selectedNgrams.value.filter(item => item !== value);
-  }
-};
-
-const handleNgramButton = (n) => {
-  selectedNgrams.value = [n.toString()];
-
-  // Uncheck all checkboxes and check only the specified n-Gram checkbox
-  document.querySelectorAll('.dropdown-menu input[type="checkbox"]').forEach(checkbox => {
-    checkbox.checked = checkbox.value === n.toString();
-  });
-};
 </script>
 
 <template>
-  <div class="ma-keywords-generator">
-    <div class="ma-header">
-      <span>Keyword Generator</span>
+  <div class="flex bg-gray-100 justify-center">
+    <div class="flex flex-col ml-2 mr-4">
+      <div class="justify-center">
+        <h1 class="text-4xl font-bold text-center text-indigo-600 mt-6">Keyword Generator</h1>
+      </div>
+      <div class="flex justify-end mb-1">
```

Comment:
Can we also use @apply directive and move long styles. See this [doc](https://tailwindcss.com/docs/functions-and-directives#apply)